### DISCUSSION TITLE
How can we pass data from Parent component to child component and update the ChildComponent UI using state observable?

### DISCUSSION THREAD

**Discussion by yuvaraj119** [arkivanov/Decompose#793] [Q&A]
I have a ParentComponent and ChildComponent
Now I have an api which gets called in parent component and then some data which will be consumed across multiple childcomponent.
Here when API results comes we have to update the ChildComponent UI with respective result.
Activity and Fragment like communication and also there could be a use case where ChildComponent wants to update some data to ParentComponent

**Comment by arkivanov** [comment]
You can just call the child component directly. Alternatively, you can use `SharedFlow` or `PublishSubject` and pass it to the child component. See the [docs](https://arkivanov.github.io/Decompose/navigation/stack/overview/#delivering-a-result-when-navigating-back).
